docs(howto): Homogenize HowTo's format across translated files (#6724)
EbookFoundation/free-programming-books: docs/HOWTO-it.md

> # How-To at a glance

Reviewer: ```suggestion
# Primi passi
```

<div align="right" markdown="1">

*[Leggilo in altre lingue](README.md#translations)*

</div>

**Benvenuto su `Free-Programming-Books`!**

Diamo il benvenuto ai nuovi collaboratori; anche a quelli che fanno la loro prima Pull Request (PR) su GitHub. Se sei uno di quelli, ecco qualche risorsa che potrebbe aiutarti:

* [Riguardante le Pull Requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) *(in inglese)*
* [Creare una pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) *(in inglese)*
* [GitHub Hello World](https://docs.github.com/en/get-started/quickstart/hello-world) *(in inglese)*
* [YouTube - GitHub Tutorial per Principianti](https://www.youtube.com/watch?v=0fKg7e37bQE) *(in inglese)*
* [YouTube - Come forkare una Repository GitHub e Inviare una Pull Request](https://www.youtube.com/watch?v=G1I3HF4YWEw) *(in inglese)*
* [YouTube - Corso accelerato di Markdown](https://www.youtube.com/watch?v=HUBNt18RFbo) *(in inglese)*

> Non esitare a fare domande; ogni collaboratore iniziato con una prima PR. So... why not join our [large, growing](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=ebookfoundation/free-programming-books) community.

Reviewer: ```suggestion
Non esitare a fare domande; ogni collaboratore ha iniziato con una prima PR. Quindi... perché non unirti alla nostra [grande e in crescita](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=ebookfoundation/free-programming-books) community?
```

<details align="center" markdown="1">

> <summary>Click to see the growth users vs. time graphs.</summary>

Reviewer: ```suggestion
<summary>Clicca per visualizzare i grafici temporali sulla crescita degli utenti.</summary>
```

[![EbookFoundation/free-programming-books's Contributor over time Graph](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorOverTime&repo=ebookfoundation/free-programming-books)](https://www.apiseven.com/en/contributor-graph?chart=contributorOverTime&repo=ebookfoundation/free-programming-books)

[![EbookFoundation/free-programming-books's Monthly Active Contributors graph](https://contributor-overtime-api.apiseven.com/contributors-svg?chart=contributorMonthlyActivity&repo=ebookfoundation/free-programming-books)](https://www.apiseven.com/en/contributor-graph?chart=contributorMonthlyActivity&repo=ebookfoundation/free-programming-books)

> NOTE: Contribution spikes use to match with the [Hacktoberfest event](https://hacktoberfest.digitalocean.com) dates.

Reviewer: ```suggestion
NOTA: I picchi corrispondono alle date dell'[evento Hacktoberfest](https://hacktoberfest.digitalocean.com).
```

</details>

> Anche se sei un collaboratore esperto in ambito open source, queste sono cose che potrebbero crearti dei problemi. Una volta che hai inviato la tua PR, ***GitHub Actions* avvierà un *linter*, spesso trovando piccoli problemi di spaziatura o di ordinamento alfabetico**. Se ottieni una spunta verde, tutto è pronto per una revisione, ma se non è così clicca su "Details" sotto il check che è stato fallito, analizza l'errore, risolvilo e aggiungi un commit alla PR.

Reviewer: ```suggestion
Anche se sei un collaboratore esperto in ambito open source, queste sono cose che potrebbero crearti dei problemi. Una volta che hai inviato la tua PR, ***GitHub Actions* avvierà un *linter*, spesso trovando piccoli problemi di spaziatura o di ordinamento alfabetico**. Se ottieni una spunta verde, tutto è pronto per una revisione, in caso contrario così clicca su "Details" sotto il check fallito, analizza l'errore e aggiungi un commit risolutivo alla PR.
```

> In fine, se non sei sicuro che la risorsa che vuoi aggiungere è appropiata a `Free-Programming-Books`, leggi le linee guida su [CONTRIBUTING](CONTRIBUTING-it.md). ([translations](README.md#translations))

Reviewer: ```suggestion
In fine, se non sei sicuro che la risorsa che vuoi aggiungere sia appropriata a `Free-Programming-Books`, leggi le linee guida su [CONTRIBUTING](CONTRIBUTING-it.md). ([traduzioni](README.md#translations))
```
